src/QuickMarcEditor/QuickMarcEditor.js

```diff
@@ -102,15 +132,26 @@ const QuickMarcEditor = ({
     );
 
     const end = (
-      <Button
-        buttonStyle="primary mega"
-        disabled={saveFormDisabled}
-        id="quick-marc-record-save"
-        onClick={confirmSubmit}
-        marginBottom0
-      >
-        <FormattedMessage id="stripes-acq-components.FormFooter.save" />
-      </Button>
+      <>
+        <Button
```

Comment:
Button to Save and continue should only be visible when you edit a record, but hidden when you derive or create a new record
Let's add a condition that `action === QUICK_MARC_ACTIONS.EDIT`